programs/psinode/main.cpp

```diff
@@ -87,7 +87,7 @@ std::string to_string(const native_service& obj)
 struct autoconnect_t
 {
    static constexpr std::size_t max   = std::numeric_limits<std::size_t>::max();
-   std::size_t                  value = max;
+   std::uint64_t value = max;
```

Comment:
`max` and `value` should have the same type

Reply:
Done.